File: atividade_nicolas/alunos/nicolas_g1/Kelson/palavras.js
import { input,print } from "../io_utils.js";

function main(){
    const frase = input("Digite uma frase de no minimo 80 e no maximo 180 caracteres: ")
    while(80>frase.length || frase.length<180)frase = input("A frase deve ter no minimo 80 caracteres: ")
    palavras(frase)
}
function palavras(frase){
    const palavras = frase.split(" ")
    for (let palavra of palavras){
        print(palavra)
    }
    print(palavras.length)
}
main()

File: atividade_nicolas/alunos/nicolas_g1/igor/palavras.dart
import '../io_utils.dart';

void main(List<String> args) {
  var inp = input("Insira uma frase com 80 a 180 caracteres: \n -> ");
  var qtd = inp.length;
  while(qtd < 80 || qtd > 180){
    inp = input("A frase deve ter de 80 a 180 caracteres:\n -> ");
  }

  var quantity = inp.split(" ").length;
  print("A frase contem $quantity palavras.");

}